pkg/lwip/Makefile.include

```diff
@@ -17,6 +17,12 @@ PSEUDOMODULES += lwip_udp
 PSEUDOMODULES += lwip_udplite
 PSEUDOMODULES += lwip_sock_async
 
+#These flags enable calculating ICMP chksum in LWIP software.
+#Without these flags RIOT device respond with ICMP chsum equal to 0, and
+#cannot be successfully "pinged" from Windows hosts.
+CFLAGS +=-DCHECKSUM_GEN_ICMP=0
```

Comment:
This is strange. According to https://www.nongnu.org/lwip/2_1_x/group__lwip__opts__checksum.html#ga2291ec5bec0a551545da6d5f9f9316b2 this would disable the generation of ICMP checksums, rather than enabling it.

I haven't looked at the code yet, but I kind of doubt that this is such an obvious bug in lwIP that `CHECKSUM_GEN_ICMP` is checked for being zero to generate the checksum. I fear that there is some other bug lurking that only gets hidden by the flags. But as said: I didn't look at the code, so these are just guesses.

Reply:
Yes, these flags are very weird but I found them in a some forum post concerning micro-controllers and similar issue - now after a more than month I cannot find this page :( . 

When I used this solution, my understanding is that used in Nucleo board ethernet controller calculate IP checksum in hardware when received from micro-controller packet set this field to 0. I try to check this in board documentation.